homeassistant/components/command_line/cover.py

```diff
@@ -66,7 +76,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     if not covers:
         _LOGGER.error("No covers added")
-        return False
+        return
```

Comment:
Unless I am mistaken, `setup_platform` shouldn't be returning a value.

Reply:
Its called here and its never checked so `None` is correct https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_platform.py#L156